eth2/attester/src/lib.rs

```diff
@@ -1,10 +1,12 @@
 pub mod test_utils;
```

Comment:
I had planned on leaving the eth2 crate alone and building a miniature, `Attester` in the validator client. 

It seems you have got it compiling. @paulhauner - Is this going to mess with any of your other harness if `attester` and `BlockProducer` gets modified?